eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -28,61 +29,73 @@ parameters:
     Linux_Python38:
       OSVmImage: 'ubuntu-18.04'
       PythonVersion: '3.8'
-
+  CloudConfigurations:
+    AzureCloud:
+      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
 jobs:
-  - job: ${{ parameters.JobName }}
-    variables:
-      skipComponentGovernanceDetection: true
-      CoverageArg: --disablecov
+  - ${{ each cloudConfig in parameters.CloudConfigurations }}:
+    - job: ${{ parameters.JobName }}
+      variables:
+        skipComponentGovernanceDetection: true
+        CoverageArg: --disablecov
+
+      timeoutInMinutes: ${{ parameters.TestTimeoutInMinutes }}
+      strategy:
+        maxParallel: ${{ parameters.MaxParallel }}
+        matrix: ${{ parameters.Matrix }}
+      continueOnError: false
 
-    timeoutInMinutes: ${{ parameters.TestTimeoutInMinutes }}
-    strategy:
-      maxParallel: ${{ parameters.MaxParallel }}
-      matrix: ${{ parameters.Matrix }}
-    continueOnError: false
+      pool:
+        vmImage: $(OSVmImage)
 
-    pool:
-      vmImage: $(OSVmImage)
+      steps:
+        - template: /eng/common/pipelines/templates/steps/bypass-local-dns.yml
 
-    steps:
-      - template: /eng/common/pipelines/templates/steps/bypass-local-dns.yml
+        - ${{ if eq(parameters['AllocateResourceGroup'], 'true') }}:
```

Comment:
I think you can delete this whole step since the same code gets run in New-TestResources.ps1 via deploy-test-resources.yml

Reply:
I think for now we will leave it because not all the libraries are set up to use the New-TestResources.ps1 script. Most of our languages use the management plane libraries.

Reply:
I remembered it actually does its own condition under the hood anyway: https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/pipelines/templates/steps/bypass-local-dns.yml#L6